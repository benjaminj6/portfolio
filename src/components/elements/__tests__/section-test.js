/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import { Section } from '../'

test('Section snapshot', () => {
  const tree = shallow(<Section />)
  expect(tree).toMatchSnapshot()
})
