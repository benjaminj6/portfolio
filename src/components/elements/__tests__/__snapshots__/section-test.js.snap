// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<section class="jsx-2426183947 jsx-2426183947 Section">
  <JSXStyle
    styleId="2426183947"
    css=".jsx-2426183947{padding:2rem 1rem;}"
  >
  </JSXStyle>
</section>

`;
